platform/windows/crash_handler_windows_signal.cpp

```diff
@@ -164,6 +164,9 @@ extern void CrashHandlerException(int signal) {
 	int64_t image_file_base = get_image_base(_execpath);
 	data.offset = image_mem_base - image_file_base;
 
+	if (FileAccess::exists(_execpath + ".debugsymbols")) {
+		_execpath = _execpath + ".debugsymbols";
+	}
```

Comment:
So I just realized - while running into it - that this can lead to loading wrong debugsymbols if you have a `.debugsymbols` file laying around from a previous build.

This can happen if you build with `separate_debug_symbols=yes` once, and then not afterwards, or if the debugsymbols are bigger than 2 GiB (which forces separate debug symbols) for a build, and then dip back under 2 GiB afterwards (which makes it default to keeping debug symbols in the binary).

These shouldn't be too frequent scenarios but this still seems brittle, and especially annoying if the main `.exe` actually contains debugsymbols but this piece of code forces it to load invalid symbols from another build.